### DISCUSSION TITLE
Accessing `Painter::max_texture_side`

### DISCUSSION THREAD

**Discussion by DusterTheFirst** [emilk/egui#1387] [Q&A] [answered]
Is there a way for UI code or even another thread to be able to access the platform specific `Painter::max_texture_side()` function so that code can change how it allocates textures if they are above the platform specific maximum. As it stands, `ctx.load_texture` will just cause a panic in the next redraw of egui if the requested texture is too large which can not be recovered from.

**Comment by emilk** [comment, marked as answer]
It can be found in `ctx.input().max_texture_side`
